lib/chef/azure/commands/enable.rb

```diff
@@ -112,31 +113,38 @@ def configure_chef_only_once
 
       load_settings
 
-      # Write validation key
-      File.open("#{bootstrap_directory}/validation.pem", "w") do |f|
-        f.write(@validation_key)
-      end
-
-      # Write client.rb
-      File.open("#{bootstrap_directory}/client.rb", "w") do |f|
-        f.write(override_clientrb_file(@client_rb))

```

Comment:
Can you remove `def override_clientrb_file(client_rb)` method If its not used anywhere ?